JSON.cpp

```diff
@@ -49,30 +59,39 @@ std::map<std::string, std::string> JSON::parseFromIstream(std::istream & instrea
 			{
 				line = "";
 			}
-			content.insert(std::pair<std::string, std::string>(key, value));
+			std::variant<std::string, int, double> mapvalue = value;
+			if (vartype == integer)
+			{
+				mapvalue = std::stoi(value);
+			}
+			else if (vartype == doubl)
+			{
+				mapvalue = std::stod(value);
+			}
+			content.insert(std::pair<std::string, std::variant<std::string, int, double>>(key, mapvalue));
 		}
 
 	}
 	
 	return content;
 }
 
-std::map<std::string, std::string> JSON::parseFromFile(const std::string & fname)
+std::map<std::string, std::variant<std::string, int, double>> JSON::parseFromFile(const std::string & fname)
```

Comment:
ha ilyen sok helyen benne van, akkor egy using-gal amugy  lehet egyszerusiteni.

Reply:
@hegyhati 
Egyszerűsítettem rajta a `using`-gal. Ha megfelel, akkor részemről mehet be master-be.